distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -140,6 +140,8 @@ type Table
          Select columns with the same names as the ones provided.
 
              table.select_columns (By_Column [column1, column2])
+
+       Icon: select-column
```

Comment:
If this is modified, then I think the analogous one in `Standard/Table` should be modified too, as these functions should be kept consistent, including the documentation.